packages/crdt/src/crdt/merkle.ts

```diff
@@ -141,7 +150,13 @@ export function prune(trie: TrieNode, n = 2): TrieNode {
 
   // Prune child nodes.
   for (let k of keys.slice(-n)) {
-    next[k] = prune(trie[k], n);
+    const node = trie[k];
+
+    if (!node) {
+      throw new Error(`TrieNode for key ${k} could not be found`);
+    }
+
+    next[k] = prune(node, n);
```

Comment:
This is a small functional change. We expect `trie[k]` to have a value. If it does not have a value - that is very unexpected and very bad (thus throw exception).